go.mod

```diff
@@ -136,7 +136,7 @@ require (
 	golang.org/x/crypto v0.21.0 // indirect
 	golang.org/x/exp v0.0.0-20230713183714-613f0c0eb8a1 // indirect
 	golang.org/x/mod v0.12.0 // indirect
-	golang.org/x/net v0.21.0 // indirect
+	golang.org/x/net v0.23.0 // indirect
```

Comment:
rather than change this generated indirect block I think it would be better to add our explicit require block here:  https://github.com/kubevirt/containerized-data-importer/blob/a2bc007da371c1c3fef8470f65e29cf53e1b8fda/go.mod#L5-L66

Reply:
I may be missing something but `make deps-update` keeps removing that change and adding it back to the indirect blok.

Reply:
ah okay cool